src/commands/generateComponent.js

```diff
@@ -37,7 +40,7 @@ function initGenerateComponentCommand(args, cliConfigFile, program) {
   // Component command action.
 
   componentCommand.action((componentNames, cmd) =>
-    componentNames.forEach((componentName) => generateComponent(componentName, cmd, cliConfigFile))
+    componentNames.forEach((componentName) => generateComponent(componentName, cmd, cliConfigFile, isFlat))
```

Comment:
We don't need to pass `isFlat` to `generateComponent` for the reason mentioned above.
```suggestion
    componentNames.forEach((componentName) => generateComponent(componentName, cmd, cliConfigFile))
```